hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorTest.java

```diff
@@ -28,82 +26,13 @@
 import org.junit.runner.RunWith;
 
 import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertTrue;
 
 /**
  * Test for different error conditions.
  */
 @RunWith(HazelcastSerialClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class SqlErrorTest extends SqlErrorAbstractTest {
-    @Test
-    public void testTimeout() {
```

Comment:
Why we don't want to test it anymore?

Reply:
Every SQL query is baked with a light job, which already have timeout test.
Also, I didn't find any specific exception that could help to determine timeout in SQL scope.